src/zeit/push/__init__.py

```diff
@@ -17,6 +17,7 @@
   channel-news News
   urbanairship-audience-group subscriptions
   mobile-buttons file://{fixtures}/mobile-buttons.xml
+  push-payload-templates data/payload-templates
```

Comment:
Hat jetzt mit dem Code nichts zu tun :), aber ich fänds glaub ich cool wenn der Ordner in Production irgendwas mit "push" und/oder "urban airship" heißen würde. Denn `payload` ist ja allgemein "Nutzlast", da wird nicht direkt klar, worums geht. `/data/urbanairship-templates` vielleicht?